src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Tag/Tag.jsx

```diff
@@ -5,16 +5,38 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {adaptSelection} from './Tag.utils';
 import {FieldPropTypes} from '~/EditPanel/FormDefinitions/FormData.proptypes';
+import {useApolloClient} from 'react-apollo-hooks';
+import {getSuggestionsTagsQuery} from './Tag.gql-queries';
+import {useContentEditorContext} from '~/ContentEditor.context';
 
 const Tag = ({field, id}) => {
     const {t} = useTranslation();
+
+    const client = useApolloClient();
+
+    const {site} = useContentEditorContext();
+
     const adaptOptions = options => (
         options.map(data => ({
             value: data,
             label: data
         }))
     );
 
+    const suggestTags = async inputValue => {
+        let variables = {
+            prefix: inputValue,
+            limit: '100',
+            startPath: '/sites/' + site
+        };
+
+        const val = await client.query({query: getSuggestionsTagsQuery, variables: variables});
+
+        return val.data.tag.suggest.map(element => {
```

Comment:
missing safety check on val.data undefined.
in case the query do not return data, we should return an empty array